staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -119,7 +120,7 @@ func (cfg *Config) Complete() CompletedConfig {
 }
 
 // New returns a new instance of CustomResourceDefinitions from the given config.
-func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget) (*CustomResourceDefinitions, error) {
+func (c completedConfig) New(serviceResolver webhook.ServiceResolver, authResolverWrapper webhook.AuthenticationInfoResolverWrapper, delegationTarget genericapiserver.DelegationTarget) (*CustomResourceDefinitions, error) {
```

Comment:
move these into the config itself. I don't see a deeper reason to have have a non-standard plumbing path for these two.